projects/core/src/user/store/actions/customer-coupon.action.spec.ts

```diff
@@ -0,0 +1,309 @@
+import {
+  CUSTOMER_COUPONS,
+  SUBSCRIBE_CUSTOMER_COUPON_PROCESS_ID,
+  UNSUBSCRIBE_CUSTOMER_COUPON_PROCESS_ID,
+  CLAIM_CUSTOMER_COUPON_PROCESS_ID,
+} from '../user-state';
+import {
+  loadMeta,
+  failMeta,
+  successMeta,
+  resetMeta,
+} from '../../../state/utils/loader/loader.action';
+import { StateEntityLoaderActions } from '../../../state/utils/index';
+import {
+  CustomerCoupon,
+  CustomerCouponSearchResult,
+  CustomerCouponNotification,
+  CustomerCoupon2Customer,
```

Comment:
This kind of name is not suggested

Reply:
it's import model name, the same as the model defined.